jormungandr/src/explorer/graphql/mod.rs

```diff
@@ -1328,6 +1520,30 @@ impl Query {
         Transaction::from_id(id, context).await
     }
 
+    /// get all current tips, sorted (descending) by their length
+    async fn tips(&self, context: &Context) -> Vec<Branch> {
```

Comment:
We use `tip` in the node protocol for the currently selected branch.
Maybe use `branch_heads` or just `branches`, considering the return type is `Vec<Branch>`?

Reply:
So, now that you bring that up... I guess what I'm calling `Branch` in the explorer/api is more like what's called `Ref` in the ledger. 

I'm not entirely convinced that name makes sense for the public api, though.

And also I don't think that queries should be limited to heads.

But maybe it makes sense to call the thing `Ref`, `StateRef`, or something like that (not *commit* unfortunately).

Reply:
`Ref` is OK in internal code, but for the public explorer API we should stick to the nomenclature used elsewhere. I think "braches" are close in meaning to the concept used in git (and git has a `HEAD` alias for what we mean by the "tip").